book/security.rst

```diff
@@ -428,6 +429,7 @@ If you'd like to load your users via the Doctrine ORM, that's easy! See
 
 .. _book-security-encoding-user-password:
 .. _c-encoding-the-users-password:
```

Comment:
Because of the single quote, the current anchor is `c-encoding-the-user-s-password` (see https://github.com/symfony/symfony-docs/pull/4755/files#diff-1c49199976c01f0f51b7fa2a54dbba30R430).